templates/compose-main.tmpl

```diff
@@ -62,12 +62,14 @@ services:
         command: server
 
     jaeger:
-        image: docker-private.infra.cloudera.com/cloudera_thirdparty/jaegertracing/all-in-one:1.13
+        image: jaegertracing/all-in-one:latest
```

Comment:
should be an internal repo

Reply:
How can I set to the latest version with internal repo?

Reply:
we should avoid latest version as well.

Reply:
I guess it requires dps-k8s changes as well. right ?

Reply:
it is now available as docker-private.infra.cloudera.com/cloudera_thirdparty/jaegertracing/all-in-one:1.14